Packs/RecordedFuture/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Recorded Future Intelligence",
     "description": "Recorded Future App, this pack is previously known as 'RecordedFuture v2'",
     "support": "partner",
-    "currentVersion": "1.7.3",
```

Comment:
I wouldn't consider this a feature/functionality add, and perhaps it should only increase the revision number i.e. 1.7.4 instead of 1.8.0

Reply:
I changed the version number to match